tracer/src/Datadog.Trace/Iast/IastModule.cs

```diff
@@ -196,6 +197,20 @@ public static IastModuleResponse OnSqlQuery(string query, IntegrationId integrat
         }
     }
 
+    public static IastModuleResponse OnNoSqlMongoDbQuery(string query, IntegrationId integrationId)
+    {
+        try
+        {
+            OnExecutedSinkTelemetry(IastInstrumentedSinks.NoSqlMongoDbInjection);
+            return GetScope(query, integrationId, VulnerabilityTypeName.NoSqlMongoDbInjection, OperationNameNoSqlMongoDbInjection, Always);
+        }
+        catch (Exception ex)
+        {
+            Log.Error(ex, "Error while checking for NoSql injection.");
```

Comment:
NIT: since we will probably have other nosql injections besides mongo, maybe we should specify in the error message that this is a mongo NoSql injection